internal/controllers/nft_controller.go

```diff
@@ -541,3 +542,45 @@ func (nc *NFTController) handleEnqueueCommand(c *fiber.Ctx, commandPath string)
 
 	return c.JSON(CommandResponse{RequestID: subTaskID})
 }
+
+type NFTGrowth struct {
+	TotalNFTs        int64   `json:"totalNFTs"`
+	GrowthPercentage float64 `json:"growthPercentage"`
+}
+
+// GetNFTImage godoc
+// @Description Returns current and difference of nft from today to last week.
+// @Tags        nfts
+// @Produce     json
+// @Success     200
+// @Failure     404
+// @Router      /vehicle/claimed-vehicles [get]
+func (nc *NFTController) GetClaimedVehicles(c *fiber.Ctx) error {
+
+	currentTime := time.Now()
+
+	lastWeek := currentTime.AddDate(0, 0, -7)
+
+	// Checking both that the nft exists and is linked to a device.
+	lastWeekNFT, err := models.VehicleNFTS(models.VehicleNFTWhere.UserDeviceID.IsNotNull(),
+		models.VehicleNFTWhere.TokenID.IsNotNull(),
+		qm.And("updated_at > $1", lastWeek)).Count(c.Context(), nc.DBS().Reader)
```

Comment:
may be able to do this entirely in sql `qm.And("updated_at > current_date - 7")`